services/app-web/src/common-code/data/statePrograms.json

```diff
@@ -832,6 +812,21 @@
                     "id": "6d1f77e3-1e97-4f57-9da0-e49358ab97e2",
                     "fullName": "SoonerCare Choice",
                     "name": "SoonerCare Choice"
+                },
+                {
+                    "id": "",
+                    "fullName": "SoonerSelect Dental",
+                    "name": "SoonerSelect Dental"
+                },
+                {
+                    "id": "",
```

Comment:
Annoying gotcha here -  we need to also manually add the `id` field when adding new programs to the list. Looks like we forgot to include this in the howto-update-state-programs doc as well, could you add it there please  as well?